src/nrf52811/nrf52811.cmake

```diff
@@ -57,7 +57,7 @@ if(OT_CFLAGS MATCHES "-pedantic-errors")
     string(REPLACE "-pedantic-errors" "" OT_CFLAGS "${OT_CFLAGS}")
 endif()
 
-set(OT_UART_BAUDRATE 115200 CACHE STRING "UART Baud rate. It must be a pre-defined
```

Comment:
The https://github.com/openthread/openthread/pull/11432 only changes the RCP baud rate. The default baud rates of CLI and NCP are not changed. They are still using the 115200 baud rate. We need to use different baud rates for different build targets.